github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -414,13 +413,9 @@ presubmits:
         - automation/test.sh
         env:
         - name: TARGET
-          value: k8s-1.20
+          value: k8s-1.20-sig-storage
         - name: KUBEVIRT_CGROUPV2
           value: "true"
-        - name: KUBEVIRT_E2E_SKIP
-          value: Multus|SRIOV|GPU|Macvtap|MediatedDevices
-        - name: KUBEVIRT_STORAGE
-          value: rook-ceph-default
```

Comment:
Why is this removed? Most of the hotplug tests that are relevant to cgroups v2 need rook-ceph in the cluster (they are skipped otherwise)

Reply:
`KUBEVIRT_STORAGE` var is set accordingly  in https://github.com/kubevirt/kubevirt/blob/314b515473a3720dce35f5477c6dbfdc7d544e5c/automation/test.sh#L52

when the `TARGET` is `k8s-*-sig-storage`

Reply:
Thanks!